### ISSUE TITLE
Blur not filling an entire window on HiDPI screen with scaling

### ISSUE THREAD

**Issue by osleg** [tsujan/Kvantum#321]
When using any transparent theme with blur it doesn't blurring an entire window (see screenshot)
![blurissue](https://user-images.githubusercontent.com/207721/50960484-76461b00-1500-11e9-837d-5ce778e85a2d.png)

That happens with any transparent/blurred theme and any application window.
If I disable scaling it works Ok

Operating System: Arch Linux 
KDE Plasma Version: 5.14.5
Qt Version: 5.12.0
KDE Frameworks Version: 5.53.0
Screen: 3840x2160
Scaling: 1.5

**Comment by tsujan** [comment]
I don't have an hdpi screen but can't see this with a scale factor > 1 and a dual screen (2 x 1920 x 1080). However, I'l look into the code.

Please answer these questions:

(1) Do you have the latest git Kvantum? If not, please upgrade and test again.

(2) Does it also happen with an integer scale factor (like 2)?

**Comment by osleg** [comment]
Kvantum version: 0.10.8
It doesn't happen with integer scale factor 
![screenshot_20190110_190210](https://user-images.githubusercontent.com/207721/50964448-4439b680-150a-11e9-962c-9ed10d62fa22.png)

You should be able to see it if you set scale factor 1.5 and re login.
Eg if I have scale factor set to 1, apply kvantum theme and set scale to 1.5 the background is ok and fills an entire application window.
Yet after log out - log in it getting broke again.
Same goes the other way. If I have 1.5 -> apply kvantum -> see blur broken -> set scale to 1 -> blur still broken -> log out -> log in -> blur is ok

**Comment by osleg** [comment]
Here is screenshot.
Logged in to plasma with scale x2, open dolphin(left), change scale to 1.5 and open another instance of dolphin (right)
![screenshot_20190110_190604](https://user-images.githubusercontent.com/207721/50964690-e6599e80-150a-11e9-97e0-9f42f131784c.png)
Now if I'll log out and log in with scale 1.5 it will be broken again

**Comment by osleg** [comment]
Here is screenshot.
Logged in to plasma with scale x2, open dolphin(left), change scale to 1.5 and open another instance of dolphin (right)
![screenshot_20190110_190604](https://user-images.githubusercontent.com/207721/50964690-e6599e80-150a-11e9-97e0-9f42f131784c.png)
Now if I'll log out and log in with scale 1.5 it will be broken again.

As a workaround  for now I don't do scaling at all but force DPI to 144 which allows me to use kvantum without issue.
Yet this creates issues in some other places/applications so the workaround is not completely good :(

**Comment by osleg** [comment]
Indeed with git branch it works as expected, thank you :)

**Comment by tsujan** [comment]
Thanks for testing!

I found a problem with menu and toolbar icons when factor==1.5; will fix it soon.

Do you see artifacts?

**Comment by osleg** [comment]
The only artifacts I see atm are the ones that comming from Qt/KDE itself, horizontal lines in Kate and Konsole like in the ss below. But that has nothing to do with kvantum afaik.

![screenshot_20190110_204645](https://user-images.githubusercontent.com/207721/50969304-fed0b580-1518-11e9-87e9-386aec0220fb.png)

Other than everything looks pretty solid.

**Comment by tsujan** [comment]
Yes, but I think if you move the mouse cursor over Dolphin's items, you might see them. If not, you're lucky ;)

I tried my best to remove them but they still exist. I might attack this problem again later.

**Comment by tsujan** [comment]
Only the vertial lines are artifacts. The doted box is called the focus rectangle and belongs to the theme; it shows the focused item.
